modules/extension/xsd/xsd-gml2/src/main/java/org/geotools/gml2/bindings/GMLEncodingUtils.java

```diff
@@ -260,31 +276,39 @@ public XSDTypeDefinition createXmlTypeFromFeatureType(SimpleFeatureType featureT
             if ( attribute instanceof GeometryDescriptor ) {
                 Class binding = attribute.getType().getBinding();
                 if ( Point.class.isAssignableFrom( binding ) ) {
```

Comment:
No, we need inheritance checks here, not simple equality (otherwise I would have used it instead of isAssignableFrom).